### PR TITLE
Dala 17 frontend sfdr questionnaire view page

### PR DESCRIPTION
**Pull request by MarcinStanczyk7**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [x] At least one E2E Test exists testing the new feature
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [x] The new version is deployed to the dev server "dev1" using this branch
  - [x] It's verified that this version actually is the one deployed (check gitinfo for branch name and commit id!)
  - [x] The new feature is manually used/tested/observed on dev server
  - [x] All implemented Social Logins have been tested manually in the UI
- [x] If any work on the UI is to be merged, those changes were also documented in the Figma

### CODE REVIEW

File: dataland-frontend/tests/e2e/specs/sfdr/DataIntegrity.ts [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+            cy.get("button.p-row-toggler").eq(0).click();
+            cy.get("table.p-datatable-table")
+              .find(`span:contains(${sfdrData.social!.general!.fiscalYearEnd!})`)
+              .should("exist");
+          });
+        });
+      });
+
+      cy.get("table.p-datatable-table").find(`span.link`).click();
+      cy.request("GET", "/test_pdf.pdf").then((response) => {
```
Comment (SiegfriedSobkowiak) [inline dataland-frontend/tests/e2e/specs/sfdr/DataIntegrity.ts:119]: as discussed lets remove the files until we actually have the document upload included

File: dataland-frontend/src/components/resources/frameworkDataSearch/DataModelsTranslations.ts [hunk 2]
```diff
@@ -129,9 +129,122 @@ export const lksgKpiNameMappings = {
   employeeUnder18: "Employee Under 18",
   codeOfConductRiskManagementTopics: "Code Of Conduct Risk Management Topics",
   responsibilitiesForTheEnvironment: "Responsibilities For The Environment",
+  fiscalYear: "Fiscal Year",
```
[the remark this answers is not in the document]
Comment (MarcinStanczyk7) [reply dataland-frontend/src/components/resources/frameworkDataSearch/DataModelsTranslations.ts]: done

File: dataland-frontend/tests/e2e/specs/sfdr/DataIntegrity.ts [hunk 3]
```diff
@@ -0,0 +1,149 @@
+import { describeIf } from "@e2e/support/TestUtility";
+import { uploader_name, uploader_pw } from "@e2e/utils/Cypress";
+import { getKeycloakToken } from "@e2e/utils/Auth";
+import { FixtureData } from "@e2e/fixtures/FixtureUtils";
+import { Configuration, SfdrDataControllerApi, SfdrData } from "@clients/backend";
+import { uploadCompanyAndSfdrDataViaApi, uploadOneSfdrDataset } from "@e2e/utils/SfdrUpload";
+import { getPreparedFixture, UploadIds } from "@e2e/utils/GeneralApiUtils";
+import { generateSfdrData } from "../../fixtures/sfdr/SfdrDataFixtures";
```
[the remark this answers is not in the document]
Comment (MarcinStanczyk7) [reply dataland-frontend/tests/e2e/specs/sfdr/DataIntegrity.ts]: done

File: dataland-frontend/src/components/resources/frameworkDataSearch/sfdr/SfdrPanel.vue [hunk 4]
```diff
@@ -0,0 +1,140 @@
+<template>
+  <div v-if="waitingForData" class="d-center-div text-center px-7 py-4">
+    <p class="font-medium text-xl">Loading Sfdr Data...</p>
```
[the remark this answers is not in the document]
Comment (MarcinStanczyk7) [reply dataland-frontend/src/components/resources/frameworkDataSearch/sfdr/SfdrPanel.vue:3]: done

File: dataland-frontend/src/components/pages/ViewSfdr.vue [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+        <div class="col-12">
+          <SfdrPanel :companyId="companyID" />
+        </div>
+      </div>
+    </template>
+    <div v-if="loading" class="col-12 text-left">
+      <h2>Checking if Sfdr data available...</h2>
+    </div>
+    <div v-if="!loading && receivedSfdrDataIds.length === 0" class="col-12 text-left">
+      <h2>No Sfdr data</h2>
```
[the remark this answers is not in the document]
Comment (MarcinStanczyk7) [reply dataland-frontend/src/components/pages/ViewSfdr.vue]: done. Also in Lksg
" <h2>Checking if LkSG data is available...</h2> "

File: dataland-frontend/src/components/pages/ViewSfdr.vue [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  <ViewFrameworkBase :companyID="companyID" dataType="sfdr" @updateDataId="handleReceivedListOfDataIds">
+    <template v-if="receivedSfdrDataIds.length > 0">
+      <div class="grid">
+        <div class="col-12">
+          <SfdrPanel :companyId="companyID" />
+        </div>
+      </div>
+    </template>
+    <div v-if="loading" class="col-12 text-left">
+      <h2>Checking if Sfdr data available...</h2>
```
[the remark this answers is not in the document]
Comment (MarcinStanczyk7) [reply dataland-frontend/src/components/pages/ViewSfdr.vue]: done

File: dataland-frontend/src/components/general/CompanyDataTable.vue [hunk 7]
```diff
@@ -94,7 +129,7 @@ export default defineComponent({
       type: Array,
       default: () => [],
     },
-    dataDatesOfDataSets: {
+    DataDateOfDataSets: {
```
[the remark this answers is not in the document]
Comment (MarcinStanczyk7) [reply dataland-frontend/src/components/general/CompanyDataTable.vue]: done. changed to dataDateOfDataSets

**Pull request merged**